### ISSUE TITLE
Change alarm sound

### ISSUE THREAD

**Issue by theonlygusti** [theonlygusti/alarm#29]
Originally a request from @waxfingers, they'd like:

> the ability of choosing and changing the beep sound. I know that it's as easy as replacing the beep.wav in the bundle folder by any other wav file, but maybe it is there an easier way of implementing the substitution.

**Comment by theonlygusti** [comment]
@waxfingers what would you want to be able to change the sound to ?

**Comment by waxfingers** [comment]
A song,  a loop, for instance... Imagine one creates several alarms: a couple reminders, 1 wake up, 1 "timer". It would be useful if we could **assign** a diferent sound feedback to each or to a group. Maybe - thinking out loud - could be there several beep options _b1 b2 b3 b4 b5_ (users could also place their own) and when creating the alarm we could assign one of those options. Hope it makes sense =)

**Comment by theonlygusti** [comment]
@waxfingers yes that makes sense. What about defining a sound that points at a file?

    alarm setsound sheeran ~/Music/Thinking\ Out\ Loud.mp4
 
then to use it

    alarm in 12h with sound sheeran

Looks ok?

**Comment by theonlygusti** [comment]
Or just use the sound,

    alarm in 12h with sound ~/Music/Thinking\ Out\ Loud.mp4

**Comment by waxfingers** [comment]
> alarm in 12h with sound ~/Music/Thinking\ Out\ Loud.mp4

That would be perfect for choosing an one _time alarm_. Dunno if can be done, but for practcality (perhaps readibility) and speed I imagine stablishing say 5 different (set)sounds and calling them with as simple an argument as possible; e.g. 

> alarm b3 in 12h Feed teacher in the basement

Of something of the sort, sorry zero code knowledge =)
